akka-docs/rst/intro/what-is-akka.rst

```diff
@@ -4,9 +4,9 @@
  What is Akka?
 ###############
 
-**Scalable real-time transaction processing**
+**«resilient elastic distributed real-time transaction processing»**
```

Comment:
it is not a sentence, so my syntax analyzer does not grant it capitalization ;-)